packages/flutter_tools/lib/src/commands/apk.dart

```diff
@@ -170,6 +226,7 @@ class ApkCommand extends FlutterCommand {
       List<ArtifactType> artifactTypes = <ArtifactType>[
         ArtifactType.androidIcuData,
         ArtifactType.androidClassesDex,
+        'TODO(mpcomplete): we need to upload new artifacts',
```

Comment:
I need to fix this before checking in. These artifacts are downloaded from cloud storage. The ones currently in storage won't work with this new method, so I need new ones.